src/bytes.ts

```diff
@@ -0,0 +1,41 @@
+const empty = new Uint8Array(0)
+
+const toHex = (d: Uint8Array): string => d.reduce((hex, byte) => hex + byte.toString(16).padStart(2, '0'), '')
```

Comment:
Does typescript make any distinction between arrow functions and `function` declarations when it comes to simple cases like this? would this come out exactly the same if we did `function toHex (d: Uint8Array): string { ...`?
One reason for switching all of these is consistency, you can see the legacy of multiple authors with different opinions in here, this one in particular was mostly Mikeal and where you see these kinds of functions with `function` it's mostly me! As long as we're doing bulk editing, maybe we should do a tiny bit of consistency work.

Reply:
What about
```ts
export function foo() {}
export function bar() {}
```
vs
```ts
function foo() {}
function bar() {}

export {foo, bar}
```

Reply:
```js
export function foo() {}
```

seems preferable to me, no need to zip around in the file working out what's exported and what isn't.

Reply:
changed